### DISCUSSION TITLE
Chakra UI MenuItem align right on large screens

### DISCUSSION THREAD

**Discussion by iansamz** [chakra-ui/chakra-ui#6313] [Q&A]
I have Menu Item aligned to left on lg screen:
<img width="253" alt="Screenshot 2022-07-13 at 11 52 30" src="https://user-images.githubusercontent.com/15368874/178692851-a5bdf6f2-ef95-4c0a-b6db-3e1b0761b334.png">
I want to align it to the right as it is in sm screen:
<img width="274" alt="Screenshot 2022-07-13 at 11 54 06" src="https://user-images.githubusercontent.com/15368874/178693197-12ce1280-6a9b-4a7a-a516-6e752d2189e5.png">

**Comment by jTanG0506** [comment]
Use `<Menu placement="top-end">`
